packages/vhd-lib/src/Vhd/VhdSynthetic.js

```diff
@@ -19,6 +19,7 @@ export class VhdSynthetic extends VhdAbstract {
     // TODO: set parentLocatorEntry-s in header
     return {
       ...vhd.header,
+      parentLocatorEntry: [...rootVhd.header.parentLocatorEntry],
```

Comment:
Why do you need to copy it?

Reply:
I don't want to modify the data of any sub vhds of a synthetic

Reply:
That's what I thought, but the objects in this array are still passed by reference.

Reply:
Either pass it directly or use `lodash.cloneDeep`.